overrides/default.nix

```diff
@@ -466,6 +466,22 @@ lib.composeManyExtensions [
         }
       );
 
+      # prevent collisions
+      flake8-debugger = super.flake8-debugger.overridePythonAttrs (
+        old: {
+          postInstall = ''
+            rm -f $out/lib/python3.9/site-packages/pyproject.toml
+          '';
+        }
+      );
+      flake8-print = super.flake8-print.overridePythonAttrs (
+        old: {
+          postInstall = ''
+            rm -f $out/lib/python3.9/site-packages/pyproject.toml
+          '';
+        }
+      );
+
```

Comment:
How can this be done better? I opted to go with python3.9 as that's what's in nixpkgs-21.11, but it's far from perfect.

Feel free to throw out of the PR.

Reply:
This is the error I see without this override:

```
error: collision between `/nix/store/cwm9fdp3w2zk7p50xab5q3dmklrbv62m-python3.9-flake8-print-4.0.0/lib/python3.9/site-packages/pyproject.toml' and `/nix/store/gcbxi1fqzkbn49qzr7c3lcbp44slpnqd-python3.9-flake8-debugger-4.0.0/lib/python3.9/site-packages/pyproject.toml'
```

Reply:
I feel like we should add a hook that cleans up any files dumped directly in the site-packages directory.

Reply:
Yea that would be very handy!

Reply:
See #551

Reply:
Fantastic, I'll rebase this PR once #551 is merged.